#include <exception>
int main () {
  return 0;
}
